app/src/ui/add-repository/add-existing-repository.tsx

```diff
@@ -18,28 +22,45 @@ interface IAddExistingRepositoryProps {
 
 interface IAddExistingRepositoryState {
   readonly path: string
-  readonly isGitRepository: boolean | null
+  readonly gitRepository: boolean
```

Comment:
Can we revert the name change here to `isGitRepository`, seems a lil' bit clearer to me.

I'd also love it if we could document these two (`isGitRepository` and `showNonGitRepositoryWarning`) to explain the differences between them